### ISSUE TITLE
Support <annotationProcessors> configuration element

### ISSUE THREAD

**Issue by sewe** [jbosstools/m2e-apt#60]
There exist annotation processors which are not declared in `META-INF/services/javax.annotation.processing.Processor`. These processors need to be explicitly selected using the `maven-compiler-plugin`’s [`<annotationProcessors>` configuration element](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#annotationProcessors) (the equivalent of `javac`’s `-processor <class>`).

AFAICT, this option is not supported by `m2e-apt` yet, making it impossible to configure annotation processing for such processors.

**Comment by sewe** [comment]
FWIW, I am aware that the JDT UI doesn’t seem to support explicit processor selection akin to `-processor` at the moment. I’ve already filed [a bug report](https://bugs.eclipse.org/bugs/show_bug.cgi?id=528704) for that.

**Comment by fbricon** [comment]
Yeah sorry, we can't work around JDT APT for that. Maybe @brychcy will be interested in fixing it upstream ;-)

**Comment by mickaelistria** [comment]
m2e-apt's code is now included in https://github.com/eclipse-m2e/m2e-core , please consider reporting issue to https://github.com/eclipse-m2e/m2e-core/issues if it's still relevant.
